index.page.tsx

```diff
@@ -113,7 +112,7 @@ export default function Index() {
           <div className="col-lg-6 mx-auto text-center pl-0 pr-0">
             <div className="d-flex flex-column-reverse">
               <h1 className="mb-10">
-                {translate('The Blockchain')}
+                {translate('Blockchain Ledger')}
```

Comment:
A blockchain is a distributed ledger, so suggest keeping the heading as is.
```suggestion
                {translate('The Blockchain')}
```